plugins/modules/keycloak_client.py

```diff
@@ -900,6 +1007,8 @@ def main():
         # they are not specified
         if client_param == 'protocol_mappers':
             new_param_value = [dict((k, v) for k, v in x.items() if x[k] is not None) for x in new_param_value]
+        elif client_param == 'authentication_flow_binding_overrides' or client_param == 'authenticationFlowBindingOverrides':
```

Comment:
Just as a matter of personal preference (feel free to ignore), I find the form:
```suggestion
        elif client_param in ['authentication_flow_binding_overrides', 'authenticationFlowBindingOverrides']:
```
more readable. YMMV